dashboard/lib/logic/task_grid_filter.dart

```diff
@@ -173,6 +183,10 @@ class TaskGridFilter extends FilterPropertySource {
       return false;
     }
 
+    if ((!_allProperties['showBringUp']?.value) && qualifiedTask.pool == 'luci.flutter.staging') {
```

Comment:
Let's add `isFlaky` (or `isBringup`) to `QualifiedTask` instead of using a magic `pool` value as a proxy.

Reply:
Done!